operator/gitops/argocd/pipeline-service/metrics-exporter/servicemonitor_openshiftconsole_rbac.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I believe this is redundant, as the openshift-pipelines operator is given access to the tekton metrics which are in the same namespace, but I did this because
- to show @ramessesii2 what exactly is meant by "adding RBAC"
- make an explicit statement that the exporter wants to expose stats to openshift-monitoring, vs. just piggy-backing on top of what  the tekton operator does 


that said, once @ramessesii2 signs off that he sees this and know understand what I am getting at, I'm fine with removing this redundancy if reviewers prefer I do